home/views.py

```diff
@@ -46,4 +49,71 @@ def students(request):
             students_add_form.save()
             return redirect(reverse('students'))
         else:
-            return HttpResponseBadRequest('Некорректно заполнены в форме')
+            return HttpResponseBadRequest('Некорректно'
+                                          ' заполнены данные в форме')
+
+
+class StudentUpdateMain(View):
+    """
+    Redirect /student-update/ page to /students-info/
+    #:param request: output 'students-info.html'
+    #:return: prints /students-info/ page
+    """
+
+    def get(self, request): # noqa
+        return redirect(reverse('students_info'))
+
+
+class StudentUpdate(View):
+    """
+    This page updated information of students
+    #:param request:
+    #:param id: id of student in database
+    :return: updated students info page
+    """
+
+    def get_student(self, id): # noqa
+        return Student.objects.get(id=id) # noqa
```

Comment:
better use in this case `get_object_or_404`
see: https://docs.djangoproject.com/en/3.1/topics/http/shortcuts/#get-object-or-404

Reply:
fixed